turn_by_turn/xtrack_line.py

```diff
@@ -1,16 +1,37 @@
 """
-XTRACK
-------
-
-This module provides functions to convert tracking results from an ``xtrack`` Line
-(or its internal ParticlesMonitor elements) into the standardized ``TbtData`` format
-used throughout ``turn_by_turn``.
+XTRACK_LINE
+-----------
+
+This module provides functions to convert tracking results from an ``xtrack.Line``
+into the standardized ``TbtData`` format used by ``turn_by_turn``.
+
+Prerequisites for using ``convert_to_tbt``:
+  1. The input ``Line`` must contain one or more ``ParticlesMonitor`` elements
+     positioned at each location where turn-by-turn data is required (e.g., all BPMs).
+     A valid monitor setup involves:
+       • Placing a ``xt.ParticlesMonitor`` instance in the line's element sequence
+         at all the places you would like to observe.
+       • Configuring each monitor with identical settings:
+           - ``start_at_turn`` (first turn to record, usually 0)
+           - ``stop_at_turn`` (The total number of turns to record, e.g., 100)
+           - ``num_particles`` (number of tracked particles)
+     If any monitor is configured with different parameters, ``convert_to_tbt`` 
+     will either find no data or raise a inconsistency error.
+  2. Before conversion, you must:
+       • Build particles with the desired initial coordinates
+         (using ``line.build_particles(...)``).
+       • Track those particles through the line for the intended number of turns
+         (using ``line.track(..., num_turns=num_turns)``).
+  
+Once these conditions are met, pass the tracked ``Line`` to ``convert_to_tbt`` to
+extract the data from each particle monitor into a ``TbtData`` object.
```

Comment:
This is __nice__. Let's check the doc build keeps a nice formatting of this, not sure if `rst` is ok with that. If not we adapt, but this is good.